steering/elections/2025/README.md

```diff
@@ -0,0 +1,82 @@
+# 2025 Istio Steering Committee election guide
+
+The Steering Committee is composed of nine (9) Contribution Seats and four (4) Community Seats. The role of this election is to elect the four Community Seats to the Istio Steering Committee for a one-year term.
+
+## Background
+
+The Istio Steering Committee oversees the administrative aspects of the project, including governance, marketing and community health. Steering exists to allow the Technical Oversight Committee to exclusively focus on the technical aspects of the project.
+
+Some direct responsibilities of steering members to consider as you are deciding whether to run or who to vote for:
+
+- Establishing and enforcing principles to guide the Istio project.
+- Fostering an environment for a healthy and happy community of developers, contributors, vendors, and users.
+- Defining, evolving, and defending a Code of Conduct.
+- Setting marketing and advocacy direction for the project; establishing a publishing schedule and vetting content, encouraging and assisting community members in creating content for conferences, fostering an ecosystem of vendors.
+- Providing neutral mediation as appropriate to try to resolve non-technical disputes that arise as part of the project.
+
+## Schedule
+
+| Date               | Event                                                               |
+|--------------------|---------------------------------------------------------------------|
+| February 12        | Announcement of Election and publication of voters.yaml             |
+| February 23        | Candidate bios and voting exception forms due by 2359 UTC (5pm PDT) |
+| February 24        | Election begins via Elekto UI                                       |
+| March 9            | Election closes at 2359 UTC (5pm PDT)                               |
+| week of March 10   | Winners announced                                                   |
+
+## Candidacy process
+
+Eligibility for candidacy is defined in the [Steering Committee charter](../../CHARTER.md) as a [project member](../../../ROLES.md#member) who does not work for a Company that will hold a Contribution Seat during the upcoming term.
+
+Members from **Solo.io**, **Microsoft**, **Huawei**, **Google** and **Tetrate** are therefore **ineligible** to stand in this election.
+
+If you want to stand for election, open a PR against the [istio/community repository](https://github.com/istio/community) to include
+your candidate profile in the `/steering/elections/2025` folder, with the file named in the format `candidate-yourname.md`. We have included [a template file](nomination-template.md) as an example. This profile should include:
```

Comment:
Should the format be "candidate-githubid.md"?

Reply:
I forget if it matters but it looks like it should